sources/packages/backend/apps/api/src/route-controllers/student-scholastic-standings/student-scholastic-standings.institutions.controller.ts

```diff
@@ -105,6 +107,15 @@ export class ScholasticStandingInstitutionsController extends BaseController {
     @UserToken() userToken: IInstitutionUserToken,
   ): Promise<PrimaryIdentifierAPIOutDTO> {
     try {
+      const application =
+        await this.applicationService.processScholasticStanding(
+          locationId,
+          applicationId,
+        );
+      payload.data.applicationOfferingStartDate =
+        application.currentAssessment.offering.studyStartDate;
+      payload.data.applicationOfferingEndDate =
+        application.currentAssessment.offering.studyEndDate;
```

Comment:
Please perform the validations mentioned in https://github.com/bcgov/SIMS/pull/3766/files#r1790758314 here.